tests/Casts/StringTest.php

```diff
@@ -27,5 +29,11 @@ public function testString(): void
 
         self::assertIsString($model->stringContent);
         self::assertEquals("Losing hope, don't mean I'm hopeless And maybe all I need is time", $model->stringContent);
+
+        $now = now();
+        $model->update(['stringContent' => now()]);
```

Comment:
You must use the variable to prevent time-related issue.
```suggestion
        $model->update(['stringContent' => $now]);
```